engine/node_latex.cc

```diff
@@ -44,6 +44,11 @@ namespace MathDAG
     string::size_type ss;
     if ((ss=nm.find_first_of("_^"))!=string::npos)
       return mathrm(nm.substr(0, ss)) + nm[ss] + "{"+mathrm(nm.substr(ss+1))+"}";
+      
+    // process % chars
+    string::size_type pc;
+    if ((pc=nm.find_first_of("%"))!=string::npos)
+      return mathrm(nm.substr(0, pc)) + "\\" + nm[pc] + mathrm(nm.substr(pc+1));      
```

Comment:
This only modifies the first % found in the string. Shouldn't you be quoting all of them? regex_replace would be the appropriate tool.

Reply:
I'm becoming suspicious of line 46 too. Was that me?